server/spec/requests/api/v1/wishes_spec.rb

```diff
@@ -24,20 +27,64 @@
             expect(json.size).to eq(5)
           end
         end
-        response '404', 'interval not found' do
+        response '404', 'User not found' do
           let(:api_user_id) { 'invalid' }
           run_test! do
             expect(json['message']).to match(/Couldn't find ApiUser with 'id'=#{api_user_id}/)
           end
         end
       end
+    end
+
+    path '/api/v1/api_users/{api_user_id}/wishes/{id}' do
+
+      get 'Displays a Wish for a particular user' do
+        tags 'List a Wish for a particular user'
+        consumes 'application/json'
+        parameter name: :api_user_id, :in => :path, :type => :string
+        parameter name: :id, :in => :path, :type => :string
+
+        response '200', 'list wish' do
+          run_test! do
+            expect(json["goal"]).not_to eq(nil)
+          end
+        end
+        response '404', 'Wish not found' do
+          let(:id) { 'invalid' }
+          run_test! do
+            # expect(json['message']).to match(/Couldn't find Wish with 'id'=#{id}/)
+          end
+        end
+      end
+    end
+
+
 
+    path '/api/v1/profile/wishes/' do
+      get 'Display all  wishes of user you are logged in as' do
+        tags 'Display all wishes of a user you are logged in as'
+        consumes 'application/json'
+        parameter name: :authorization, :in => :header, :type => :string 
+
+        response '422', 'no token' do
+          let(:authorization){nil}
+          run_test! do
+            expect(json["message"]).to eq("Missing token")
+          end
+        end
+
+        response '200', 'Display wishes current user logged in' do
+          run_test! do
+            expect(json.count).to eq(5)
+          end
+        end
+      end
 
 
-      post 'Create a wish' do
-        tags 'Create a wish for an API User'
+      post 'Create a wish for user you are logged in as' do
+        tags 'Create a wish for user you are logged in as'
         consumes 'application/json'
-        parameter name: :api_user_id,  :in => :path, :type => :string
+        parameter name: :authorization, :in => :header, :type => :string      
```

Comment:
It's possible to use the ":" too:

```suggestion
        parameter name: :authorization, in: :header, type: :string      
```

Reply:
Agree and updaetd